src/test/java/org/commcare/formplayer/tests/SavedFormDefTest.java

```diff
@@ -86,7 +88,7 @@ public void testIrrelevantDataIsNotIncludedInSubmission() throws Exception {
                 "requests/submit/submit_hidden_value_form.json", sessionId);
         assertEquals("success", submitResponseBean.getStatus());
 
-        Mockito.verify(this.submitServiceMock).submitForm(formSession.getInstanceXml(false),
-                formSession.getPostUrl());
+        Mockito.verify(this.submitServiceMock).submitForm(any(),
```

Comment:
This change greatly reduces the value of this test. I think the whole point of the test was to check that the XML submitted matched expectations which it no longer does. I think you'll need to use `argThat` with a custom argument matcher to validate the XML in the mulipart body.

Reply:
Thanks for flagging, decided to use an argument captor instead - https://github.com/dimagi/formplayer/pull/1209/commits/ca4949eaab8ab1b0a827e7d7f0dbcc548f891c37